antora-playbook.yml

```diff
@@ -61,7 +61,7 @@ asciidoc:
     image-puller-name: Kubernetes Image Puller
     image-puller-name-short: Image Puller
     image-puller-repository-name: kubernetes-image-puller
-    image-puller-serviceaccount-name: k8s-image-puller
+    # image-puller-serviceaccount-name: kubernetes-image-puller
```

Comment:
variable removed and hardcoded lower down? um... ok?

Reply:
1) the value is `kubernetes-image-puller` not  `k8s-image-puller`
2) this value is the same for Che and CRW, so why use the attribute there?

I just left it there, commented out, and put the value hardcode in, since there is no need for the attribute.
@nickboldt

Reply:
makes sense.